build/docker/Dockerfile-base-python

```diff
@@ -0,0 +1,5 @@
+FROM python:3.8.0-alpine3.10 as alpine
```

Comment:
not really, 3.8 is just a bit later, do you want me to change it to 3.7?

Reply:
I think it would be wroth keeping it in sync with what's in .plzconfig https://github.com/thought-machine/dracon/pull/11/files#diff-5742873e8fe29cd5ab0c86470a3dcea4R18